runner/runner.go

```diff
@@ -34,7 +34,7 @@ const (
 )
 
 func (p ProcessState) IsDone() bool {
-	return p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
+	return p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT || p == BADREQUEST
```

Comment:
As discussed in person:

1) BADREQUEST should probably not be IsDone, because a retry could work
2) BADREQUEST is weird, and maybe we should just return an error instead of a ProcessStatus...